packages/grafana-ui/src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from '@emotion/css';
+import React, { useCallback } from 'react';
+
+import { GrafanaTheme2, ThemeSpacingTokens } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+
+type AlignItems =
+  | 'stretch'
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'baseline'
+  | 'start'
+  | 'end'
+  | 'self-start'
+  | 'self-end';
+
+type JustifyContent =
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly'
+  | 'start'
+  | 'end'
+  | 'left'
+  | 'right';
+
+type Direction = 'row' | 'row-reverse' | 'column' | 'column-reverse';
+
+type Wrap = 'nowrap' | 'wrap' | 'wrap-reverse';
+
+interface FlexProps {
+  gap?: ThemeSpacingTokens;
+  alignItems?: AlignItems;
+  justifyContent?: JustifyContent;
+  direction?: Direction;
+  wrap?: Wrap;
+  height?: number;
```

Comment:
height in pixels feels awkward here , why height and no width (and why height and no minHeight/minWidth etc) . Why pixels vs grid units is also a bit strange :)